libs/sdk/tests/IJsonLinkedDataProofSuite.spec.ts

```diff
@@ -1,13 +1,14 @@
-import { PublicKey } from 'verifiablecredentials-crypto-sdk-typescript-keys';
+import base64url from 'base64url';
+import { KeyType, OkpPublicKey, PublicKey } from 'verifiablecredentials-crypto-sdk-typescript-keys';
 /*---------------------------------------------------------------------------------------------
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 import { LongFormDid, CryptoBuilder, IJsonLinkedDataProofSuite, JoseBuilder, KeyUse, KeyStoreOptions } from '../lib';
 import SuiteJcsEd25519Signature2020 from '../lib/suites/SuiteJcsEd25519Signature2020';
 
-describe('SuiteJcsEd25519Signature2020', ()=> {
-    fit('should sign and verify a payload', async () => {
+describe('SuiteJcsEd25519Signature2020', () => {
```

Comment:
Just FYI, even thought this format "looks like linked data"... it does not actually use ANY JSON-LD :)

https://w3c-ccg.github.io/ld-cryptosuite-registry/#jcs-ed25519-signature-2020
https://identity.foundation/JcsEd25519Signature2020/